client/app/house/components/MembersManagementModal.tsx

```diff
@@ -0,0 +1,149 @@
+    const data = {
+      userId,
+      houseId,
+      isAdmin,
+    }
+
+    onUpdate(data)
+  }
+
+  const handleOpenAlert = (userId: number) => {
+    setRemovedMemberId(userId)
+    setIsOpenModal(false)
+    setIsOpenAlert(true)
+  }
+
+  return (
+    <Dialog open={isOpenModal} onOpenChange={() => setIsOpenModal(false)}>
+      <DialogContent
+        // To prevent closing modal when clicking toast
+        onPointerDownOutside={(e) => {
+          if (
+            e.target instanceof Element &&
+            e.target.closest('[data-radix-toast-announce-exclude]')
+          ) {
+            e.preventDefault()
+          }
+        }}
+      >
+        <DialogHeader className='pb-2'>
+          <DialogTitle className='text-xl'>Members management</DialogTitle>
+        </DialogHeader>
+        <div className='hide-scrollbar flex max-h-96 flex-col overflow-scroll'>
+          <div className='flex items-center justify-end gap-12 pb-2 text-lg font-semibold'>
+            <span>Role</span>
+            <span>Remove</span>
+          </div>
+          {houseMembers.map((houseMember) => (
+            <div
+              className='flex border-separate border-spacing-y-1 items-center justify-between border-b border-slate-400 py-2'
+              key={houseMember.id}
+            >
+              <div className='flex items-center gap-2'>
+                <MdAccountCircle
+                  className={`text-4xl ${colorMap[houseMember.icon.toString()]}`}
+                  size={42}
+                />
+                <span className='text-md'>{houseMember.name}</span>
+              </div>
+              <div className='flex items-center gap-12 pr-4'>
+                {userId !== houseMember.id && (
+                  <>
+                    <div className='flex items-center gap-4'>
+                      <Checkbox
+                        className='h-5 w-5 text-primary'
+                        checked={houseMember.isAdmin}
+                        disabled={isUpdating}
+                        onClick={() =>
+                          handleUpdateAdmin(
+                            houseMember.id,
+                            !houseMember.isAdmin,
+                          )
+                        }
+                      />
+                      <span>admin</span>
+                    </div>
+                    <div className='flex items-center justify-center'>
+                      <Button
+                        variant='destructiveOutline'
+                        size='icon'
+                        className='flex-shrink-0'
+                        type='button'
+                        aria-label={`Remove roommate ${houseMember.name}`}
+                        onClick={() => handleOpenAlert(houseMember.id)}
+                      >
+                        <FiX size={18} />
+                      </Button>
+                    </div>
+                  </>
+                )}
```

Comment:
Two things! 
1. I think we should show the disabled button and checkbox instead of not showing anything if the user is the person who currently logged in. 
2. Why the first person is always not showing? If you want a user to control other people (not the user self), I think the logic should be connected getMe.

Reply:
Fixed!
https://github.com/HAK2024/fairy-share/pull/93/commits/b282b8a7e297f3136deb61ef816dea9e749b07a4
https://github.com/HAK2024/fairy-share/pull/93/commits/37d401c6c9064c363eae61b235da2e1229308517